aws/logs_monitoring/enhanced_lambda_metrics.py

```diff
@@ -451,24 +452,11 @@ def parse_metrics_from_report_log(report_log_line):
     else:
         tags.append("cold_start:false")
 
-    # if cold_start:
-    if regex_match.group(INIT_DURATION_METRIC_NAME):
-        metric_point_value = float(regex_match.group(INIT_DURATION_METRIC_NAME))
-        # Multiply by 1/1000 to convert ms to seconds
-        metric_point_value *= METRIC_ADJUSTMENT_FACTORS[INIT_DURATION_METRIC_NAME]
-
-        initial_duration = DatadogMetricPoint(
-            "{}.{}".format(
-                ENHANCED_METRICS_NAMESPACE_PREFIX, INIT_DURATION_METRIC_NAME
-            ),
-            metric_point_value,
-        )
-
-        initial_duration.add_tags(tags)
-
-        metrics.append(initial_duration)
-
     for metric_name in METRICS_TO_PARSE_FROM_REPORT:
+        # check if init duration is present
```

Comment:
Nits -> `# check whether the metric, e.g., init duration, is present in the REPORT log`